Resources/Prototypes/_CM14/Recipes/Construction/Graphs/structures/barricade_plasteel_door.yml

```
@@ -0,0 +1,32 @@
+- type: constructionGraph
+  id: BarricadePlasteelDoorGraph
+  start: start
+  graph:
+  - node: start
+    actions:
+    - !type:DeleteEntity
+    edges:
+    - to: node_barricade_plasteel_door
+      completed:
+      - !type:SnapToGrid
+        southRotation: true
+      steps:
+      - material: Plasteel
```

YML linter is saying Plasteel isn't the correct entity ID, it should be SheetPlasteel
```suggestion
      - material: SheetPlasteel
```